crates/subspace-networking/src/behavior.rs

```diff
@@ -50,10 +50,10 @@ pub(crate) struct BehaviorConfig<RecordStore> {
     pub(crate) autonat: AutonatWrapperConfig,
 }
 
-#[derive(Debug, Clone, Copy)]
-pub(crate) struct GeneralConnectedPeersInstance;
-#[derive(Debug, Clone, Copy)]
-pub(crate) struct SpecialConnectedPeersInstance;
+// #[derive(Debug, Clone, Copy)]
+// pub(crate) struct GeneralConnectedPeersInstance;
+// #[derive(Debug, Clone, Copy)]
+// pub(crate) struct SpecialConnectedPeersInstance;
```

Comment:
remove them instead ?

Reply:
This is because there was other code commented-out below and this one was forgotten. I suggested @shamil-gadelshin to remove it from the beginning IIRC.